rcon/utils.py

```diff
@@ -102,19 +102,19 @@ def get_map_side(map_):
     "foy_warfare": "Foy",
     "foy_offensive_ger": "Foy Offensive (GER)",
     "foy_offensive_us": "Foy Offensive (US)",
-    "stmariedumont_warfare": "St Marie du Mont",
-    'stmariedumont_off_us': "St Marie Du Mont Offensive (US)",
-    'stmariedumont_off_ger': "St Marie Du Mont Offensive (GER)",
+    "stmariedumont_warfare": "Sainte-Marie-du-Mont",
```

Comment:
That's too long. Character real estate is expensive. St. is a proper abreviation for Saint / Sainte, that's what you'd see on road signs in Fance, we also use the shortcut for post and such. To be proper you coud add an e, sot "Ste"
dash are not required either and would prevent word / line breaks in some places so it's not ideal either.And finally to be totally pedantic "du" should not be capitalised as it's just a connecting word like "of the" 

See the image below. And no not all french dress like that ahaha only farmers from my grandpa's generation ;)
![image](https://user-images.githubusercontent.com/4445376/192164901-985574e8-a934-41b9-a863-af762220aa3f.png)